.github/workflows/test.yml

```diff
@@ -52,6 +52,10 @@ jobs:
         # sprinkle some test data (refreshing computed fields is done from the command handler)
         docker compose exec django python manage.py init --data --users
 
+        # test 'signalo'
+        docker compose exec django python manage.py test signalo.core
+        docker compose exec django python manage.py test signalo.edge_cases
```

Comment:
not sure, but probably `docker compose exec django python manage.py test` or `docker compose exec django python manage.py test signalo` would be enough

Reply:
btw you can add this to the quickstart section of the readme too